components/posts/posts.php

```diff
@@ -279,12 +281,13 @@ function mm_posts_output_post_title( $post, $context, $args ) {
 		return;
 	}
 
-	$link_title = mm_true_or_false( $args['link_title'] );
+	$heading_level = $args['heading_level'];
+	$link_title    = mm_true_or_false( $args['link_title'] );
 
 	if ( $link_title ) {
 
 		printf(
-			'<h1 class="entry-title" itemprop="headline"><a href="%s" title="%s" rel="bookmark">%s</a></h1>',
+			'<' . $heading_level . ' class="entry-title" itemprop="headline"><a href="%s" title="%s" rel="bookmark">%s</a></' . $heading_level .'>',
```

Comment:
Good change but I think you should move $heading_level down into the replaceable things and just `%s` in the top of the printf string, and then wrap it in esc_attr().  `mm_posts_output_postmeta_values()` and `mm_posts_output_postmeta_value()` take a similar approach but could also use esc_attr() wrapping the element name.